src/content/ui/widgets/material.md

```diff
@@ -1,8 +1,36 @@
 ---
-title: Material Components widgets
+title: Material component widgets
 short-title: Material
 description: > 
   A catalog of Flutter's widgets implementing Material 3 design guidelines.
 ---
 
+Flutter provides a variety of visual, behavioral, and motion-rich widgets
+that implement the [Material 3][] design specification.
+Material 3 is the default design language of Flutter,
+enabling you to design and build beautiful, usable apps
+that can adapt to any platform.
+
+:::secondary
```

Comment:
Doesn't secondary require an alternate title? It used to.

Reply:
It's optional :) - In this case I didn't want to add a title to reduce how much attention the aside attracted.